bedrock/mozorg/templates/mozorg/home/home-en.html

```diff
@@ -69,11 +69,20 @@ <h1>{{ self.page_title() }}</h1>
     {{ download_firefox(dom_id='download-sticky', download_location='sticky cta', button_color='mzp-t-small') }}
   {% endcall %}
 
-  {{ fxa_banner(
-    title=_('Firefox'),
-    sub_title=_('Live your life.<br> Own your life.'),
-    link_cta=_('Learn More'),
-  )}}
+  {% if l10n_has_tag('mozorg_hero_subhead') %}
```

Comment:
I think we can do without the l10n_has tag here since the page has not been translated yet. (It's English only right now except for the German page but it gets its own copy).